atomicapp/applogging.py

```diff
@@ -121,7 +121,7 @@ def setup_logging(verbose=None, quiet=None, logtype=None):
         if logging_level == logging.DEBUG:
             formatstr = '%(asctime)s - [%(levelname)s] - %(longerfilename)s - %(message)s'
         else:
-            formatstr = '%(asctime)s - [%(levelname)s] - %(filename)s - %(message)s'
+            formatstr = '[%(levelname)s] - %(filename)s - %(message)s'
 
```

Comment:
Ok, I like what you have done, but I think we should document this better as I had to dig through the logging library code to actually figure out that adding "%s" below was specifying the time format to print out. Instead of what you have can we, right here in the code, do something like this:

```
# Set a tuple of options that will be passed to the formatter. The %s will 
# tell the logging library to use seconds since epoch as the output time format.
formattup = (formatstr, "%s")
```
and then use *formattup below as your function call args.